composer.json

```diff
@@ -30,7 +30,6 @@
         "php-http/guzzle6-adapter": "^1.1",
         "php-http/httplug-bundle": "^1.10",
         "sensio/framework-extra-bundle": "^4.0",
```

Comment:
Can be removed from composer and it can be added as an annotation, https://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html

Reply:
Annotations is an alias of FrameworkExtraBundle, so it's the same.